# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/lm_sensors/no_depmod.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-02-06 20:23:17.000000000 +0200
+++ ./Makefile	2004-04-14 20:30:17.000000000 +0300
@@ -259,13 +259,7 @@
 	@echo "***    to fit your needs again."
 all :: user
 install :: all user_install
-	@echo "*** The depmod command below may generate errors. We are aware of the"
-	@echo "*** problem and are working on a solution."
-ifeq ($(DESTDIR),)
-	-/sbin/depmod -a
-else
-	-/sbin/depmod -a -b $(DESTDIR)
-endif
+	@echo "*** RockLinux: depmod -a is pending!"
 
 clean::
 	$(RM) lm_sensors-*
